tests/functional/aws-node-sdk/lib/utility/makeRequest.js

```diff
@@ -0,0 +1,87 @@
+import http from 'http';
```

Comment:
I like this utility but it's not really related to the aws-node-sdk.  Let's make this a separate directory called something like raw-node.  You will also have to create a new script for testing in the package.json, update circle.yml here and update Integration to call the new script.